### ISSUE TITLE
Analytics Implementation for Accredited Representation Management - Find a Representative (MVP)

### ISSUE THREAD

**Issue by oddball-lindsay** [department-of-veterans-affairs/va.gov-team#69230]
# Google Analytics Implementation

### Required
- **Is this request for **Google Analytics implementation** or **QA**?**

     - [x] Google Analytics implementation
     - [ ] QA

- **Who is this request for?**

    - Accredited Representation Management Team
    - PM = Lindsay Li-Smith
    - [Insert team name(s), for individual's names please use the `/support` tracker on the #vfs-platform-support channel and link this ticket.]
    
- **Provide the link(s) for Google Analytics implementation or QA**
  
    - No existing analytics yet (we are launching our MVP)
    
- **When is the launch date?**
    - Staged Rollout 12/4-12/13

- **What interactions need to be tracked?** _NOTE: [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) are automatically tracked_
    
    - Please see the KPIs in our [Find a Representative Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/accredited-representation-management/product-documentation/find-a-representative/product-outline-find-a-representative.md#measuring-success) 

## Acceptance criteria
- [ ] All appropriate issue tagging is completed
- [ ] QA has been completed
- [ ] Events have been added to `Product` and `Benefit Hub` Content Groups

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Analytics Request Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html) on Platform website for more information about the Collaboration Cycle.

**Comment by michelle-dooley** [comment]
@oddball-lindsay once those pages launch you will be able to see most of those metrics in GA reports. And then we can get you added to Domo dashboards. I don't believe there is anything to be done yet since those are pages are not yet live unless I am missing something in your request?

**Comment by oddball-lindsay** [comment]
That sounds right @michelle-dooley!  I submitted this a bit earlier as instructions mentioned
> Request this touchpoint after you've defined your KPIs and determined which user interactions would be beneficial to track **before you build your product/application.**

I do have both Domo and Google Analytics access already.  Would the next step be to circle back here after we're live, for support building a Domo dashboard and GA report, or is that up to our ARM team to build product-specific dashboards/reports?

**Comment by michelle-dooley** [comment]
@oddball-lindsay yes after you launch (or right before) you would just need to submit a [dashboard request ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=michelle-dooley&labels=analytics-insights%2C+analytics-request%2C+kpi-dashboard&projects=&template=analytics-request-kpi-dashboard.md&title=Analytics+New+KPI+dashboard+request+for+%5BTeam%5D) and we will take care of it.
